pkg/skaffold/test/custom/custom.go

```diff
@@ -54,7 +53,7 @@ func New(cfg docker.Config, wd string, ct latest.CustomTest) (*Runner, error) {
 // Test is the entrypoint for running custom tests
 func (ct *Runner) Test(ctx context.Context, out io.Writer, _ []build.Artifact) error {
 	if err := doRunCustomCommand(ctx, out, ct.customTest); err != nil {
-		return fmt.Errorf("running custom test command: %w", err)
+		return cutomTestErr(err)
```

Comment:
doesn't this wrap everything in `cutomTestErr` ?

Reply:
Yes.

Reply:
unlike in `structure.go` this will wrap all errors with `customTestErr`. Effectively you'll never get the other errorCodes. Can you add tests that check the error code for some of them?

Reply:
Removed cutomTestErr().